packages/website/docs/record-extractor.md

```diff
@@ -244,7 +268,24 @@ type CustomVariable =
     };
 ```
 
-Contains values that can be used as [`facetFilters`][15]
+Custom variables are used to [`filter your search`](/docs/DocSearch-v3#filtering-your-search), you can define them in the [`recordProps`](#indexing-content-for-faceting)
+
+## `helpers.docsearch` API Reference
```

Comment:
Can be merged as-is but belongs into official documentation not here
https://www.algolia.com/doc/tools/crawler/apis/configuration/actions/#parameter-param-recordextractor

Reply:
I did not found any mention of it on the Algolia doc so I've only added here for now. To avoid duplicate, a doc should point to the other, feel free to decide!